src/config.ts

```diff
@@ -95,6 +95,11 @@ class DiscordBridgeConfigBridge {
     public disableJoinLeaveNotifications: boolean = false;
     public disableInviteNotifications: boolean = false;
     public determineCodeLanguage: boolean = false;
+    public userBlacklist: string[] = [];
+
+    public IsUserBlacklisted(userId: string): boolean {
+        return !!userId && this.userBlacklist.some((regex) => RegExp(`^${regex}$`).test(userId));
```

Comment:
not sure if adding `^` and `$` here already is a good idea

Reply:
Like, if you just want to blacklist `:example.org$` or something, it could confuse people that they have to add `.*`